src/natcap/invest/validation.py

```diff
@@ -758,7 +803,7 @@ def validate(args, spec, spatial_overlap_opts=None):
             if warning_msg:
                 validation_warnings.append(([key], warning_msg))
                 invalid_keys.add(key)
-        except Exception:
+        except Exception as ex:
```

Comment:
Was this a linting suggestion as well? Just curious.

Reply:
oops! no this was just left over from a previous change